Consumedir keeps on restarting

This seems to be the case since a couple of day(s) - I read something you introduced the check for the consumedir if everything is setup correctly. The thing is, besides a new docker image I didn't change a thing and before it worked :-)

```
Checking integration endpoint: curl  -i -s -o /dev/null -w %{http_code} -H Docspell-Integration:t.........b http://docspell-restserver:7880/api/v1/open/integration/item/_archive
[ERROR] 404 response integration endpoint: curl  -i -s -o /dev/null -w %{http_code} -H Docspell-Integration:t.........b http://docspell-restserver:7880/api/v1/open/integration/item/_archive
```

--- a/tools/consumedir.sh
+++ b/tools/consumedir.sh
@@ -346,8 +346,8 @@ checkSetup() {
                     trace "Checking integration endpoint: $CURL_CMD $OPTS "$url""
                     status=$($CURL_CMD $OPTS "$url")
                     if [ "$status" != "200" ]; then
-                        echo "[ERROR] $status response integration endpoint: $CURL_CMD $OPTS $url"
-                        exit 1
+                        echo "[WARN] Collective '$(basename $collective)' failed the setup check."
+                        echo "[WARN] $status response, command: $CURL_CMD $OPTS $url"
                     fi
                 fi
             done
@@ -356,7 +356,7 @@ checkSetup() {
 }
 
 
-# quit here if something is not correctly configured
+# warn if something seems not correctly configured
 checkSetup
 
 if [ "$once" = "y" ]; then

